templates/index.html

```diff
@@ -181,22 +399,43 @@
     </script>
 </head>
 
-<body class="bg-black">
-    <div class="chat-body card">
-        <div class="card-body p-5">
-            <h4 class="card-title text-center text-xl font-medium"> Document Chat </h4>
-            <p class="card-text text-center text-sm" id="header"> Ask a question about the documents?
-            </p>
-            <hr class="border-gray-500 mb-5" style="margin-top: 20px;">
-            <div id="messages" class="overflow-auto" style="max-height: 500px;">
+<body>
+    <nav class="helper-nav">
+        <svg class="helper-logo" id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 3000 926.64"><defs><style>.cls-1{fill:#393e46;}.cls-2{fill:#eee;}</style></defs><g id="SvgjsG2224"><path class="cls-1" d="M486.49,81.08,81.08,312.74V776.06l405.41,231.66L891.78,776.13l.11-.18V312.74Zm0,89.18L774.79,335l-78,44.57L486.49,259.42,276.21,379.59l-78-44.61ZM657.86,446.48V597.74L525.51,522.16V370.89Zm-210.4-75.59V522.13l-132.31,75.6V446.48Zm0,525.37L159.09,731.49V401.84l78,44.57V686.88L447.46,807.1ZM354.1,664.6,486.45,589,618.83,664.6,486.49,740.2Zm459.78,66.89L525.51,896.26V807.1L735.87,686.93V446.41l78-44.57V731.49Z" transform="translate(-81.08 -81.08)"/></g><g id="SvgjsG2225"><path class="cls-2" d="M1151.86,235.14V420.41h2.06a122.91,122.91,0,0,1,104.7-59.23q68.18,0,98.83,34.44t30.65,108.82V726.91h-97.8V524.42q0-43.39-14.12-64.75t-48.56-21.35q-39.25,0-57.51,23.76t-18.25,78.18V726.91h-97.81V235.14Zm475.24,126q50.28,0,89.54,23.42t61.65,67.84q22.38,44.43,22.38,102.28,0,5.5-.69,17.91h-256.9q1.37,42.71,22.38,66.46t64.4,23.77a94,94,0,0,0,49.25-13.43q22.38-13.44,28.58-32.72h86.09q-37.87,119.85-166.68,119.84-48.9-.69-90.57-21.35t-66.46-64.4q-24.79-43.72-24.8-101.59,0-54.4,25.14-99.18t66.81-66.81a189.26,189.26,0,0,1,89.88-22Zm75.08,149.46q-6.89-39.94-25.48-57.86t-53-17.9q-35.82,0-56.48,20.31t-24.1,55.45Zm256.22-275.5V726.91h-97.81V235.14Zm277.57,126q73.7,0,117.78,51.66t44.08,139.81q0,80.6-42.36,132.25t-114,51.65q-70.26,0-106.76-53.72h-1.38V852.27h-97.8V370.82h93v45.46h1.38Q2165,361.19,2236,361.18ZM2129.9,549.9q0,52.35,22,82.65t63.37,30.31q42.71,0,63.71-30.31t21-82.65q0-53.73-22.72-84.37t-63.37-30.65q-40.63,0-62.33,30.3t-21.7,84.72ZM2621,361.18q50.28,0,89.54,23.42t61.64,67.84q22.39,44.43,22.39,102.28,0,5.5-.69,17.91H2537q1.38,42.71,22.38,66.46t64.4,23.77A94,94,0,0,0,2673,649.43q22.38-13.44,28.58-32.72h86.1Q2749.78,736.56,2621,736.55q-48.9-.69-90.57-21.35T2464,650.8q-24.79-43.72-24.8-101.59,0-54.4,25.14-99.18t66.81-66.81a189.33,189.33,0,0,1,89.89-22Zm75.07,149.46q-6.89-39.94-25.48-57.86t-53-17.9q-35.8,0-56.47,20.31T2537,510.64Zm363.67-149.46q12.39,0,21.35,3.44v90.92A165.41,165.41,0,0,0,3046,452.1q-96.42,0-96.42,114.33V726.91h-97.81V370.82h93v66.12h1.37a120.6,120.6,0,0,1,45.81-55.1q31.34-20.65,67.84-20.66Z" transform="translate(-81.08 -81.08)"/></g></svg>
```

Comment:
would be good to get these SVG's out into separate files in the future, maybe if you go to React or something later on